programs/test/cpp_dummy_build.cpp

```diff
@@ -100,6 +93,13 @@
 #include "mbedtls/x509_crt.h"
 #include "mbedtls/x509_csr.h"
 #include "mbedtls/xtea.h"
+#include "../library/bn_mul_internal.h"
```

Comment:
I don't think those should be here. The point of this program is to check that you can build a C++ program linking to the library and `#include`ing its public headers. Those headers are no longer public, so there's no reason to test that they can be `#include`d from C++ (we don't support building the library itself as C++).

Reply:
Ah okay I wasn't aware of the purpose of the file so I just made the change to make it build properly, I will remove it now.

Reply:
Done.